packages/mds-types/index.ts

```diff
@@ -140,12 +140,57 @@ export interface VehicleCountMetricObj {
   dead: number | null
 }
 
+export interface MetricBadEvents {
```

Comment:
Seems like some weird changes to be in here

Reply:
Thanks for spotting that. I had Michael's ORM branch merged in at one point. Clearly I didn't get rid of all of it.